teachertool/src/style.overrides/Button.scss

```diff
@@ -39,4 +39,16 @@
 
 .common-button.disabled {
     opacity: 0.8;
+}
+
+.rounded > .common-button.common-dropdown-button {
+    border-radius: 1rem;
+}
+
+.fail > .common-button.common-dropdown-button {
+    background-color: #FFD15C;

```

Comment:
No color literals please. Use vars. They will need to be themed for high contrast.